performance/graphite-setup.yaml

```diff
@@ -0,0 +1,102 @@
+dse_version: 6.0.11
+# Driver branch to use
+driver_oss_branch: 4.x
+# Driver dse branch to use
+driver_dse_branch: 4.x
+# Driver version identifier (used as part of graphite prefix)
+driver_version: 4.8.0
+# Driver examples branch to use
+driver_examples_branch: java-driver-4.x
+# How long to run test for
+duration: 2d
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool
+        properties:
+          product.type: dse
+          product.install.type: tarball
+          product.version: {{dse_version}}
+          cassandra.yaml:
+            hinted_handoff_enabled: false
+          datacenters:
+            datacenter1:
+              size: 3
+              workload: cassandra
+  client:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool
+        properties:
+          java.version: openjdk8
+          install.maven: true
+      - name: java_driver
+        properties:
+          oss.git.repository: git@github.com:datastax/java-driver.git
+          oss.git.branch: {{driver_oss_branch}}
+          dse.git.branch: {{driver_dse_branch}}
```

Comment:
Do we still need this?

Reply:
I don't know. I don't want to dig into debugging this now. I propose to investigate it in a separate JIRA task